src/features/people/PersonList/index.js

```diff
@@ -1,26 +1,54 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { nanoid } from "@reduxjs/toolkit";
 import { Section } from "../../../common/Section"
 import { Title } from "../../../common/Title";
-import { PersonTile } from "./PersonTile";
+import { PersonTile } from "../../../common/PersonTile";
+import { ErrorPage } from "../../../common/ErrorPage";
+import { Loader } from "../../../common/Loader";
 import { Wrapper } from "./styled";
-import { usePopularPeople } from "./usePopularPeople"
+import {
+  fetchPeople,
+  selectError,
+  selectLoading,
+  selectPeople,
+} from "./popularPeopleSlice";
+import { APIImageUrl } from "../../APIdata";
 
 export const PersonList = () => {
-  const popularPeople = usePopularPeople();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchPeople());
+  }, [dispatch])
+
+  const popularPeople = useSelector(selectPeople);
+  const loading = useSelector(selectLoading);
+  const error = useSelector(selectError);
 
   return (
     <Section>
-      <Title>Popular people</Title>
-      <Wrapper>
-        {
-          popularPeople?.data?.map((person, index) => (
-            <PersonTile
-              key={index}
-              profileImage={`https://image.tmdb.org/t/p/original${person.profile_path}`}
-              fullName={person.name}
-            />
-          ))
-        }
-      </Wrapper>
+      {error
+        ? <ErrorPage />
+        : <>
+          <Title>Popular people</Title>
+          {loading
+            ? <Loader />
+            : <Wrapper>
+              {
+                popularPeople.map(person => (
+                  <PersonTile
+                    key={nanoid()}
+                    profileImage={`${APIImageUrl}/original${person.profile_path}`}
```

Comment:
I think we can change "original" to "w185" to keep API response as light as possible 😀 
For more info about that look at:
https://developers.themoviedb.org/3/configuration/get-api-configuration

Reply:
100% agree, changed to w185 to make the response lighter